tests/02_eradiate/01_unit/scenes/atmosphere/test_heterogeneous.py

```diff
@@ -306,11 +313,92 @@ def test_heterogeneous_blend_switches(mode_mono):
         particle_layers=[ParticleLayer()],
     )
 
-    # Purely absorbing atmosphere + particle layer combination is not allowed
-    with pytest.raises(ValueError):
-        HeterogeneousAtmosphere(
-            molecular_atmosphere=MolecularAtmosphere.ussa_1976(
-                has_absorption=True, has_scattering=False
+
+def test_heterogeneous_absorbing_mol_atm(
+    mode_ckd, tmp_path, absorbing_only, scattering_only
+):
+    """
+    Phase function weights are correct when the molecular atmosphere is
+    absorbing-only and the particle layer is either absorbing-only or
+    scattering-only.
+    """
```

Comment:
You should also check collision coefficients (nonabsorbing atmosphere + particle layer extinction coefficient should be the sum of individual components).

Reply:
It seems to me that this is already adressed by `test_heterogeneous_mix_collision_coefficients`